lib/Util/Cookies.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace MailPoet\Util;
+
+use InvalidArgumentException;
+
+class Cookies {
+  const DEFAULT_OPTIONS = [
+    'expires' => 0,
+    'path' => '',
+    'domain' => '',
+    'secure' => false,
+    'httponly' => false,
+  ];
+
+  function set($name, $value, array $options = []) {
+    $options = $options + self::DEFAULT_OPTIONS;
+    $value = json_encode($value, JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES);
```

Comment:
@costasovo It's kind of a best practise I guess - it produces natural clean JSON as you would expect it (same as from JavaScript I guess). See: https://github.com/nette/utils/blob/master/src/Utils/Json.php#L32-L34

There is also `JSON_PRESERVE_ZERO_FRACTION` which is really cool to avoid unwanted float-to-int casts but that exists only since PHP 5.6.6. It's good to have JSON wrapped by some class like that one from Nette I guess. Maybe we can do it one day.